docs/git/cheat-sheet.md

```diff
@@ -0,0 +1,78 @@
+# Git Cheat Sheet
+
+The following is a cheat sheet for using [git] while developing Cardano SL
+which should contain all the git operations that someone working on Cardano
+will need at some point.
+
+### Cloning the Cardano SL repository
+```
+git clone https://github.com/input-output-hk/cardano-sl
+```
+After you change directory into that cloned repository, you may want to edit
+the `.git/config` file which has a section:
```

Comment:
Why not the following instead?
```
git remote set-url --push origin git@github.com:input-output-hk/cardano-sl
```

Do we want to focus on editing files in `.git`?

Reply:
Thanks!